apps/web/src/components/ExperienceFormFields/ExperienceHeading.tsx

```diff
@@ -0,0 +1,185 @@
+import * as React from "react";
+import { useIntl } from "react-intl";
+import {
+  Accordion,
+  DefinitionList,
+  Heading,
+  StandardAccordionHeader,
+} from "@gc-digital-talent/ui";
+
+import BookOpenIcon from "@heroicons/react/20/solid/BookOpenIcon";
+import BriefcaseIcon from "@heroicons/react/20/solid/BriefcaseIcon";
+import LightBulbIcon from "@heroicons/react/20/solid/LightBulbIcon";
+import StarIcon from "@heroicons/react/20/solid/StarIcon";
+import UserGroupIcon from "@heroicons/react/20/solid/UserGroupIcon";
+import PencilSquareIcon from "@heroicons/react/24/outline/PencilSquareIcon";
+import PlusCircleIcon from "@heroicons/react/24/outline/PlusCircleIcon";
+import { experienceTypeTitles } from "~/pages/Applications/ApplicationCareerTimelineAddPage/messages";
+
+type AccordionStates = "learn-more" | "";
+
+interface ExperienceHeadingProps {
+  edit?: boolean;
+}
+
+const ExperienceHeading = ({ edit }: ExperienceHeadingProps) => {
+  const intl = useIntl();
+  const [accordionState, setAccordionState] =
+    React.useState<AccordionStates>("");
+
+  return (
+    <div>
+      {edit ? (
+        <>
+          <Heading
+            Icon={PencilSquareIcon}
+            data-h2-margin-top="base(0)"
+            data-h2-display="base(flex)"
+            color="primary"
+          >
+            {intl.formatMessage({
+              defaultMessage: "Edit an experience",
+              id: "RzXSJ9",
+              description:
+                "Button to open modal to add a new experience to the profile",
+            })}
+          </Heading>
+          <p data-h2-margin="base(x1, 0)">
+            {intl.formatMessage({
+              defaultMessage:
+                "This form allows you to edit a specific experience, its details, and linked skills. Don’t forget, work experiences should focus on describing your part-time, full-time, self-employment, fellowship, non-profit, or internship experiences.",
+              id: "9Eh1nm",
+              description:
+                "Instructions on how to add an experience to your career timeline",
+            })}
+          </p>
+        </>
+      ) : (
+        <>
+          <Heading
+            Icon={PlusCircleIcon}
+            data-h2-margin-top="base(0)"
+            data-h2-display="base(flex)"
+          >
```

Comment:
Sorry, I did not realize this used two different `Heading`'s here. We will need `color="primary"` on this one as well.

```suggestion
          <Heading
            Icon={PlusCircleIcon}
            color="primary"
            data-h2-margin-top="base(0)"
            data-h2-display="base(flex)"
          >
```

Reply:
Nice catch